openassessment/xblock/openassessmentblock.py

```diff
@@ -127,6 +129,13 @@ class OpenAssessmentBlock(MessageMixin,
         'staff-assessment',
     ]
 
+    VALID_ASSESSMENT_TYPES_DISPLAY_NAMES = {
```

Comment:
Can you add the types from the mockups and give them more descriptive names here?